migrator/migrations/n_57_to_n_58_postgres_remove_clustercve_permission/migration.go

```diff
@@ -0,0 +1,63 @@
+package n57ton58
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/migrations/n_57_to_n_58_postgres_remove_clustercve_permission/postgres"
+	"github.com/stackrox/rox/migrator/types"
+	pkgMigrations "github.com/stackrox/rox/pkg/migrations"
+)
+
+const (
+	batchSize = 500
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: pkgMigrations.CurrentDBVersionSeqNumWithoutPostgres() + 57,
+		VersionAfter:   &storage.Version{SeqNum: 58},
+		Run: func(databases *types.Databases) error {
+			err := cleanupPermissionSets(databases.PostgresDB)
+			if err != nil {
+				return errors.Wrap(err, "updating PermissionSet schema")
+			}
+			return nil
+		},
+	}
+	clusterCVEResourceName = "ClusterCVE"
+)
+
+func cleanupPermissionSets(db *pgxpool.Pool) error {
+	ctx := context.Background()
+	store := postgres.New(db)
+	permissionSetsToInsert := make([]*storage.PermissionSet, 0, batchSize)
+	err := store.Walk(ctx, func(obj *storage.PermissionSet) error {
```

Comment:
It took me some time to realize how this magically works without specifying the table. Can we rename the package with `postgres_plugin.go` into something more specific so that it's clear that `.New()` is a local function that produces a store specializing on permission sets? Maybe `postgresPlugin.New()`?